apps/api/src/admin/dto/bccnm-ncas-update.dto.ts

```diff
@@ -79,11 +79,70 @@ export class BccnmNcasUpdateItemDTO implements BccnmNcasValidation {
   bccnmDecisionDate?: string;
 
   @ApiModelProperty({
-    description: 'BCCNM Registration Date',
+    description: 'BCCNM Full License LPN',
   })
   @IsDateString()
   @IsOptional()
-  bccnmRegistrationDate?: string;
+  bccnmFullLicenceLPN?: string;
+
+  @ApiModelProperty({
+    description: 'BCCNM Full License RPN',
+  })
+  @IsDateString()
+  @IsOptional()
+  bccnmFullLicenceRPN?: string;
+
+  @ApiModelProperty({
+    description: 'BCCNM Full License RN',
+  })
+  @IsDateString()
+  @IsOptional()
+  bccnmFullLicenceRN?: string;
+
+  @ApiModelProperty({
+    description: 'BCCNM Provisional License RN',
```

Comment:
Licence for consistency?

Reply:
Thank you! I thought I had caught all the instances where Licence was spelt License. I will fix this.